### PR TITLE
Complete the feature "conditional based on tag"

### PR DESCRIPTION
**Pull request by RTLcoil**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,7 @@ logs
 *.log
 npm-debug.log*
 config.ts
+.env
```
Comment (eitanp461) [inline .gitignore]: Why is this change necessary?

Comment (RTLcoil) [reply .gitignore]: .env file contains private information about Clodinary credentials used in UT, added to .gitignore to avoid this private information sharing by mistake.

Comment (eitanp461) [reply .gitignore]: @RTLcoil what type of information is stored there for a unit test?
Why should unit tests depend on private info?
How would the test run on CI?

Comment (RTLcoil) [reply .gitignore]: Sorry @eitanp461, for some reason my answer wasn't sent. Here it is:

1. CLOUDINARY_URL=...
2. Sorry, this .gitignore change is related with clodinary_npm, this file was copied into clodinary_angular by mistake, so this change could be omitted
3. Right now tests could be run with a simple 'npm run test-once'
'test' script cannot be used because it runs karma in watch mode without --single-run --no-auto-watch flags and will lead to CI task runner timeout

**Comment by shedar** [comment]
Requirement are not finalized. Closing for now.
